### PR TITLE
add gptq model test in pytorch hub tests

### PR DESCRIPTION
**Pull request by eaidova**
### Details:
 - *added example of gptq quantized model into torch hub tests*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: tests/model_hub_tests/torch_tests/test_hf_transformers.py [hunk 1]
```diff
@@ -286,8 +347,13 @@ def forward(self, flattened_patches, attention_mask):
             except:
                 pass
         if model is None:
-            from transformers import AutoModel
-            model = AutoModel.from_pretrained(name, torchscript=True)
+            from transformers import AutoModel, AutoConfig
+            config = AutoConfig.from_pretrained(name)
```
Comment (mvafin) [inline tests/model_hub_tests/torch_tests/test_hf_transformers.py:351]: Could you patch config in the beginning?

Comment (eaidova) [reply tests/model_hub_tests/torch_tests/test_hf_transformers.py]: moved

File: tests/model_hub_tests/torch_tests/test_hf_transformers.py [hunk 2]
```diff
@@ -325,3 +392,9 @@ def test_convert_model_precommit(self, name, type, ie_device):
     @pytest.mark.nightly
     def test_convert_model_all_models(self, name, ie_device):
         self.run(model_name=name, model_link=None, ie_device=ie_device)
+
+    def _run(self, model_name, model_link, ie_device):
+        super()._run(model_name, model_link, ie_device)
+        if self.cuda_available is not None:
+            unpatch_gptq(self.cuda_available, self.gptq_postinit)
```
[the remark this answers is not in the document]
Comment (eaidova) [reply tests/model_hub_tests/torch_tests/test_hf_transformers.py]: moved

**Review by mvafin — COMMENTED** [review]
LGTM, but I would need to run nightly tests locally to verify that nothing is broken.

File: tests/model_hub_tests/torch_tests/test_hf_transformers.py [hunk 3]
```diff
@@ -12,6 +12,61 @@
 from torch_utils import TestTorchConvertModel
 from torch_utils import process_pytest_marks
 
+def is_gptq_model(config):
+    config_dict = config.to_dict()
```
[the remark this answers is not in the document]
Comment (eaidova) [reply tests/model_hub_tests/torch_tests/test_hf_transformers.py]: fixed

**Review by mvafin — APPROVED** [review]

**Pull request merged**